middle-pgsql.cpp

```diff
@@ -1263,21 +1263,21 @@ struct pthread_thunk {
 extern "C" void *pthread_middle_pgsql_stop_one(void *arg) {
     pthread_thunk *thunk = static_cast<pthread_thunk *>(arg);
     return thunk->obj->pgsql_stop_one(thunk->ptr);
-};
+}
 } // anonymous namespace
 
 void middle_pgsql_t::stop(void)
 {
     int i;
 #ifdef HAVE_PTHREAD
-    pthread_t threads[num_tables];
+    std::vector<pthread_t> threads(num_tables);
```

Comment:
This is not legal C++. VLA's are an extension and not included in (even the latest) C++ standard. The reason for using std::vector here is not failing to compile using a standard conforming compiler (e.g. with -pedantic or without extensions).